include/proc_ui/procui.h

```diff
@@ -127,10 +127,11 @@ ProcUIDrawDoneRelease();
  * Determines whether the application is in the foreground.
  *
  * \returns
- * \c true if the application status is #PROCUI_STATUS_IN_FOREGROUND.
+ * \c true if the application status is #PROCUI_STATUS_IN_FOREGROUND or #PROCUI_STATUS_RELEASE_FOREGROUND.
```

Comment:
I'm not sure if we should document this entirely using the status, since they're not completely related.
From taking a quick look at the `proc_ui.rpl`, `InForeground` is set to true after the foreground acquired message and only set to false after a pending `DrawDoneRelease`.
Maybe something like this would work?
```suggestion
 * \c true if the application is in foreground (e.g. during #PROCUI_STATUS_IN_FOREGROUND or #PROCUI_STATUS_RELEASE_FOREGROUND).
```

Reply:
Yep that's what I saw.